### ISSUE TITLE
Ability to right and select "bypassed" results

### ISSUE THREAD

**Issue by notoriousturtle** [Quitten/Autorize#22]

**Comment by barakt11** [comment]
Just to be clear, you would like that Autorize will allow you to see only the results with "bypassed" status via right click?

**Comment by Quitten** [comment]
Just to be clear, you would like that Autorize will allow you to see only the results with "bypassed" status via right click?

**Comment by notoriousturtle** [comment]
No, I'd like to be able to select a bunch of rows in the table, then right click, and copy the selected to clipboard (to paste into Microsoft Word for example). 

It would also be good if there was an option to remove duplicate URLs, and leave uniques only. 

Thanks!

**Comment by bit4woo** [comment]
I also want to select multiple items to "copy url" or "retest" even delete, hope this function ~ thanks
